autoPlay.user.js

```diff
@@ -495,6 +499,14 @@ function MainLoop() {
 	}
 }
 
+function useAllAbilities() {
+	for(var key in ABILITIES) {
+		if(canUseAbility(ABILITIES[key])) {
```

Comment:
Oh yea, you don't even need to call `canUseAbility`, `tryUsingAbility` already performs these checks.